### ISSUE TITLE
Example in python for authentication with `DVID` layer

### ISSUE THREAD

**Issue by SridharJagannathan** [google/neuroglancer#297]
Hi @jbms ,
Currently I have a segmentation data that is only accessible via a token and is configured as a dvid datasource.
When I try to add it as a segment layer, I get the following error.

```
Traceback (most recent call last):
  File "/Users/sri/Documents/Python/neuroglancer/python/neuroglancer/sockjs_handler.py", line 52, in _on_changed
    provider = self.credentials_manager.get(parsed_key['key'], parsed_key.get('parameters'))
  File "/Users/sri/Documents/Python/neuroglancer/python/neuroglancer/credentials_provider.py", line 31, in get
    return self._providers[key](parameters)
KeyError: 'DVID'
```
I noticed that there are examples for configuring brainmaps/gc authentication here:
https://github.com/google/neuroglancer/blob/master/python/neuroglancer/default_credentials_manager.py

Can you please provide an example like this to configure for DVID from the python side. The token is already cached and would be captured from an env var.

**Comment by jbms** [comment]
As you noticed, it is necessary to implement a separate credentials provider in Python, because the normal mechanism used by the neuroglancer client for getting credentials require that the neuroglancer client be served from a permitted origin (that is the mechanism by which the authentication server prevents other websites from stealing your credentials).  Since with the Python integration the client runs on a random port on your own machine, it doesn't have a permitted origin and needs to obtain the credentials through a separate mechanism.

Currently there is no DVID credentials provider implement in Python.  But if you would like to implement one, I'd be happy to include it.

**Comment by SridharJagannathan** [comment]
Closed via #299
